docs/resources/Guild.md

```diff
@@ -94,6 +94,14 @@ Guilds in Discord represent an isolated collection of users and channels, and ar
 | HIGH      | 3       | must be a member of the server for longer than 10 minutes |
 | VERY_HIGH | 4       | must have a verified phone number                         |
 
+###### Guild NSFW Level
+| Level          | Value   |
+| -------------- | ------- |
+| DEFAULT        | 0       |
+| EXPLICIT       | 1       |
+| SAFE           | 2       |
+| AGE_RESTRICTED | 3       |
```

Comment:
I have to comment on this, but why is the order the way it is? Wouldn't it have made more sense to have the levels and corresponding values sorted from least to most NSFW here?

Like (Assuming this order is right... Can't tell which is more NSFW of Explicit and age_restricted) having the order `SAFE`, `DEFAULT` (Is this higher than safe?), `AGE_RESTRICTED`, `EXPLICIT` would've made more sense since now Safe is between two NSFW ratings, which is weird from a design perspective.

But again is this just a comment pointing this rather minor issue out... Just needed this out of my system.

Reply:
yeah, I noticed this too. It's going from `EXPLICIT` to `SAFE` and then again back to `AGE_RESTRICTED`. Seems a bit off.

Reply:
According to some chat in JDA is this due to the NSFWLevel existing before and already having DEFAULT and EXPLICIT and now SAFE and AGE_RESTRICTED where added?
That does make sense, but yeah... just feels so off...

Reply:
It's an enum, not a monotonically increasing weight